games-roguelike/FTL-gog/FTL-gog-1.6.3.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit desktop
+
+DESCRIPTION="Top-down roguelike space ship simulator"
+HOMEPAGE="https://www.gog.com/game/faster_than_light"
+SRC_URI="ftl_advanced_edition_en_1_6_3_17917.sh"
+RESTRICT="bindist fetch strip test"
+
+LICENSE="all-rights-reserved"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+IUSE=""
+
+DEPEND="app-arch/unzip"
+RDEPEND="media-libs/freetype
+	media-libs/libsdl[X,sound,opengl,video]
+	media-libs/libpng
+	sys-libs/zlib
+	virtual/opengl"
+
+S="${WORKDIR}"
```

Comment:
Use `S="${WORKDIR}/data/noarch"` instead, to simplify the paths in `src_install`.